cfn-resources/api-key/cmd/resource/resource.go

```diff
@@ -168,7 +169,7 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (handler
 		return *peErr, nil
 	}
 	// Set the roles from model
-	apiKeyInput := atlasSDK.CreateAtlasOrganizationApiKey{
+	apiKeyInput := atlasSDK.UpdateAtlasOrganizationApiKey{
```

Comment:
do you know why we were using the `create..` here?

Reply:
This was a change [released](https://github.com/mongodb/atlas-sdk-go/releases/tag/v20230201008.0.0) in the latest version of the SDK
`(*ProgrammaticAPIKeysApiService).UpdateApiKey: changed from func(context.Context, string, string, *CreateAtlasOrganizationApiKey) UpdateApiKeyApiRequest to func(context.Context, string, string, *UpdateAtlasOrganizationApiKey) UpdateApiKeyApiRequest`

Note that only the input data type was changes but the API call is correct and still made using `.UpdateApiKey`